### PR TITLE
Misc codebase changes

### PR DESCRIPTION
**Pull request by quardbreak**
* Recommendations extensions replaced with BYOND extension pack
* Updates gitignore ~~with useful scripts for people out the box (BayStation port).~~
* Add admin paralyze check to roundstart logout report (BayStation port).
* Display disposal pipes above tiles in map editors (BayStation port)

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
 	"recommendations": [
-		"gbasood.byond-dm-language-support",
-		"platymuus.dm-langclient"
+		"ss13.byond"
```
Comment (quardbreak) [inline .vscode/extensions.json:3]: this is bundled extension with the same extensions as above, plus extension for commenting blocks

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+				"$dreammaker"
+			],
+			"group": {
+				"kind": "build",
+				"isDefault": true
+			},
+			"label": "dm: build - nebula.dme"
+		}
+	]
+}
```
Comment (PsiOmegaDelta) [inline .vscode/tasks.json]: Is there any reason for including a file that's automatically generated, and interferes with people with a custom `tasks.json` file?

Comment (quardbreak) [reply .vscode/tasks.json]: Not sure but kinda yes. I lost mine one time and unable to initialize new one. This one makes shift-ctrl-b execute build compile by default without pressing what kind build to do.

Comment (quardbreak) [reply .vscode/tasks.json]: Decided to not include it.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**